Gems/AWSCore/Code/CMakeLists.txt

```diff
@@ -97,22 +100,31 @@ if (PAL_TRAIT_BUILD_HOST_TOOLS)
                 Gem::AWSCore.Editor.Static
     )
 
-    # This target is not a real gem module
-    # It is not meant to be loaded by the ModuleManager in C++
-    ly_add_target(
-        NAME AWSCore.ResourceMappingTool MODULE
-        NAMESPACE Gem
-        OUTPUT_SUBDIRECTORY AWSCoreEditorQtBin
-        FILES_CMAKE
-            awscore_resourcemappingtool_files.cmake
-        INCLUDE_DIRECTORIES
-            PRIVATE
-                Include/Private
-        BUILD_DEPENDENCIES
-            PRIVATE
-                Gem::AWSCore.Editor.Static
-    )
-    ly_add_dependencies(AWSCore.Editor AWSCore.ResourceMappingTool)
+    if (PAL_TRAIT_ENABLE_RESOURCE_MAPPING_TOOL)
+
+        # This target is not a real gem module
+        # It is not meant to be loaded by the ModuleManager in C++
+        ly_add_target(
+            NAME AWSCore.ResourceMappingTool MODULE
+            NAMESPACE Gem
+            OUTPUT_SUBDIRECTORY AWSCoreEditorQtBin
+            FILES_CMAKE
+                awscore_resourcemappingtool_files.cmake
+            INCLUDE_DIRECTORIES
+                PRIVATE
+                    Include/Private
+            BUILD_DEPENDENCIES
+                PRIVATE
+                    Gem::AWSCore.Editor.Static
+            RUNTIME_DEPENDENCIES
+                3rdParty::pyside2
+
+        )
+        ly_add_dependencies(AWSCore.Editor AWSCore.ResourceMappingTool)
+
+        ly_install_directory(DIRECTORIES Tools/ResourceMappingTool)
```

Comment:
Minor: I think this line is not required, which copies resource mapping tool file into install folder. (and at the end of the file, it does the work in anyway, as except launching the tool from editor, developer should be able to launch it by using python)

I guess you just copy from the stabilization branch PR directly?

Reply:
This part is merged from stabilization automatically without manual changes. We can address it in dev later.